notebooks/en/rag_with_unstructured_data.ipynb

```diff
@@ -0,0 +1,531 @@
+{
```

Comment:
<p>I think as of now if you just do notebook_login() there's no need to save it, it's sort of encouraged to do it that way</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>